packages/core/src/Components/Icon/Types/Icon.ts

```
@@ -1,12 +1,14 @@
+import type { ComponentName }      from '@/Shared';
 import type { IconSize }           from '@/Shared/UseSize';
 import type { HorizontalPosition } from '@/Shared/Utils';
-import type { ComponentName }      from '@/Shared/Utils/Internal';
 import type { Component }          from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Shared/Utils" (internal-type).

(perfectionist/sort-imports)